packages/pagination/src/interfaces.ts

```diff
@@ -1,25 +1,29 @@
 import { CSSObject } from 'styled-components';
 
 import { IThemeNamespace } from '@xcritical/theme';
-import { ISelectBaseTheme } from '@xcritical/select';
+import { ISelectBaseTheme, SelectProps } from '@xcritical/select';
 
 import { ButtonGroupTheme, ButtonTheme } from '@xcritical/button';
 
 
 export interface IPagination {
   currentPage?: number;
   showSizeChanger?: boolean;
+  isDisabled?: boolean;
+  showTotalInfo?: boolean;
   pageSize?: number;
   pageSizeOptions?: number[];
   visibleRange?: 3 | 5 | 7;
   total: number;
+  selectProps?: SelectProps;
```

Comment:
What is it selectProps? I do not understand what this parameter means?

Reply:
If I use showSizeChanger it renders select. I need it to be configurable.
You want me to rename it to sizeChangerProps?